packages/ketcher-core/src/domain/entities/text.ts

```diff
@@ -27,21 +27,30 @@ export enum TextCommand {
 
 export interface TextAttributes {
   // TODO: add Interface for content type
-  content?: string
+  content: string
   position?: Vec2
+  pos: Array<Vec2> | []
 }
 
 export class Text {
   content: string
-  position: Vec2
+  position?: Vec2
+  pos: Array<Vec2> | []
 
   constructor(attributes?: TextAttributes) {
+    this.pos = attributes?.pos || []
     this.content = attributes?.content || ''
     this.position = attributes?.position
       ? new Vec2(attributes.position)
       : new Vec2()
   }
 
+  setPos(coords: Array<Vec2> | null): void {
```

Comment:
I guess it'll be a bit better to set an empty array to pos if `coords` var is null